src/capture.js

```diff
@@ -441,11 +441,13 @@ Capture.ios8AndGreaterScrollFix = function(doc, callback) {
     var head = doc.getElementsByTagName('head');
     // Be extra safe and guard against `head` not existing.
     if (!head.length) {
+        callback && callback();
         return;
     }
-    var head = head[0];
 
-    var meta = document.createElement('meta');
+    head = head[0];
```

Comment:
@jansepar @fractaltheory and I were looking at this last night, and realized that we needed to change our approach a bit.

**If we're adapting the page**
- We should set the viewport tag only to `device-width`, because base.dust will eventually set it to `device-width`, so any other value will cause the iOS8 scrolling issue.

**If we're NOT adapting the page**
We should not do anything, since we'll restore the original desktop source. 

- If there was no viewport tag, the browser will implicitly create one.
- If there was one in the source, it'll be restored.